bentoml/yatai/deployment/sagemaker/serve

```diff
@@ -66,4 +66,16 @@ def _serve():
 
 
 if __name__ == '__main__':
+    container = BentoMLContainer()
+    config = BentoMLConfiguration()
+    if "BENTOML_GUNICORN_TIMEOUT" in os.environ:
+        config.override(["api_server", "port"], int(os.environ.get("BENTOML_GUNICORN_TIMEOUT")))
+    if "BENTOML_GUNICORN_NUM_OF_WORKERS" in os.environ:
+        config.override(["api_server", "workers"], int(os.environ.get("BENTOML_GUNICORN_NUM_OF_WORKERS")))
+    container.config.from_dict(config.as_dict())
+
+    from bentoml import service, yatai
+
+    container.wire(packages=[service, yatai])
+
```

Comment:
What's the best way to test this? @parano @yubozhao @bojiang

Reply:
We don't have a good way right now.  Deployment tests (e2e_tests) are running offline right now.

We need to refactor/improve the sagemaker deployment with better gunicorn supports anyway. I am fine that we continue to do the basic testings on the e2e test